vm/vm.go

```diff
@@ -924,7 +924,7 @@ func (vm *VM) Submit(
 
 		// Verify auth if not already verified by caller
 		if verifyAuth && vm.config.VerifyAuth {
-			msg, err := tx.Digest()
+			msg, err := tx.UnsignedBytes()
```

Comment:
Can we use `tx.Verify(ctx)` here since the next step is `tx.Auth.Verify(ctx, msg)` ?

Reply:
Feel free to ignore this one so it can be cleaned up when we switch to separate types for signed / unsigned txs